<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusWindow</class>
 <widget class="QWidget" name="StatusWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Peripheral monitoring</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="generatorGroup">
     <property name="title">
      <string>Generator</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ch2 Laser modulation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="ch1Label">
        <property name="text">
         <string>n/a</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ch1 bias modulation of APD:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="ch2Label">
        <property name="text">
         <string>n/a</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="ch1Status">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>n/a</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="ch2Status">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>n/a</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="APDGroup">
     <property name="title">
      <string>Photodetector</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLabel" name="uBiasLabel">
        <property name="text">
         <string>n/a</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="kLabel">
        <property name="text">
         <string>n/a</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>V</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>K:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ubias:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>DC-DC Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="dcdcStatus">
        <property name="text">
         <string>n/a</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="laserGroup">
     <property name="title">
      <string>Laser</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Power P:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="laserPLable">
        <property name="text">
         <string>n/a</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>V</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
